app/views/articles/subscriptions.html.erb

```diff
@@ -0,0 +1,65 @@
+<% title "Subscriptions #{@article.title}" %>
+
+<% data = [
+     {
+       name: "John Smith",
+       email: "john.smith@example.com",
+       timestamp: "Just now"
+     },
+     {
+       name: "Bradley Cooper",
+       email: "b.cooper_1292@yahoo.com",
+       timestamp: "1 h ago"
+     },
+     {
+       name: "Louis von Honderhoffen",
+       email: "luigi.funky.one@hey.com",
+       timestamp: "Yesterday"
+     },
+     {
+       name: "Paweł Ludwiczak",
+       email: "ludwiczakpawel@hey.com",
+       timestamp: "Last week"
+     },
+     {
+       name: "Lisa Sy",
+       email: "lisa@sy.com",
+       timestamp: "Last month"
+     },
+     {
+       name: "Bill Clinton",
+       email: "billy.c@gov.com",
+       timestamp: "3 years ago"
```

Comment:
We probably should use actual timestamps here; we'll need to include additional logic to convert them and format into a string (but I think @thepracticaldev/delightful can implement this change if necessary!).

Reply:
Cool. Yea, i think having relative dates with a tooltip showing full timestamp is the ideal state. But if we can't have that, showing full timestamp will be enough.

Reply:
@vaidehijoshi @ludwiczakpawel I'm slightly biased towards relative time because our timestamps aren't quite where we want them to be WRT timezones 😬 . We've got a couple of open issues regarding that. For this "v1", I'm thinking maybe what's most important is an author has an easy way to identify "recent" subscriptions, either way.

Reply:
@atsmith813 I should clarify: I do agree that we should format the timestamps to be relative to the user's timezone; I was more just trying to point out here that the sample data here has a string for the time, when more likely, from the backend, it will be a `created_at` timestamp...right?

Reply:
@vaidehijoshi ohhhhhh yes, I see now. Correct!